api-tests/auth-service-tests/auth-service-api.tests.json

```
@@ -0,0 +1,284 @@
+{
+	"info": {
+		"_postman_id": "e262d0c2-b563-4461-b948-807ee579f43e",
+		"name": "AuthServiceAPITest",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json",
+		"_exporter_id": "36861410"
+	},
+	"item": [
+		{
+			"name": "Login",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							" pm.test(\"Get login URL should return 200 with valid url redirect\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"    const responseJson = pm.response.json();\r",
+							"    const urlRedirect = responseJson.urlRedirect;\r",
+							"    const queryParams = urlRedirect.split(\"?\")[1].split(\"&\");\r",
+							"    console.log(queryParams);\r",
+							"    const responseType = queryParams.find(q => q.includes(\"response_type\")).split(\"=\")[1];\r",
+							"    const scope = queryParams.find(q => q.includes(\"scope\")).split(\"=\")[1];\r",
+							"    const clientId = queryParams.find(q => q.includes(\"client_id\")).split(\"=\")[1];    \r",
+							"    const state = queryParams.find(q => q.includes(\"state\")).split(\"=\")[1];\r",
+							"    const nonce = queryParams.find(q => q.includes(\"nonce\")).split(\"=\")[1];\r",
+							"    const redirectUrl = queryParams.find(q => q.includes(\"redirect_uri\")).split(\"=\")[1];\r",
+							"    pm.expect(responseType).to.be.eq(\"CODE\");\r",
+							"    pm.expect(scope).to.be.eq(\"openid\");\r",
+							"    pm.expect(clientId).to.be.a(\"string\");\r",
+							"    pm.expect(state).to.be.a(\"string\");\r",
+							"    pm.expect(nonce).to.be.a(\"string\");\r",
+							"    pm.expect(redirectUrl).to.be.a(\"string\");\r",
+							"    pm.environment.set(\"STATE\", state);\r",
+							"    pm.environment.set(\"NONCE\", nonce);\r",
+							"    pm.environment.set(\"REDIRECT_URL\", urlRedirect);\r",
+							"});\r",
+							""
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"auth": {
+					"type": "noauth"
+				},
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{CHECKOUT_HOST}}/checkout/auth-service/v1/auth/login?recaptcha=03AFcWeA5W",
```

```suggestion
					"raw": "{{CHECKOUT_HOST}}/checkout/auth-service/v1/auth/login?recaptcha=test",
```

🚲 use an explicit test recaptcha token, just to make clear there is no recapcha validation logic in those api calls